nodes/mode.js

```diff
@@ -0,0 +1,88 @@
+module.exports = function(RED) {
+  const fetch = require('node-fetch');
+
+  function castHubitatValue(dataType, value) {
+    switch(dataType) {
+      case "STRING":
+        return value;
+      case "ENUM":
+        return value;
+      case "NUMBER":
+        return parseFloat(value);
+      case "BOOL":
+        return value == "true";
+      default:
+        console.warn("Unable to cast to dataType. Open an issue to report back the following output:");
+        console.warn(dataType);
+        console.warn(value);
+        return value;
+    }
+  }
```

Comment:
You can remove it

Reply:
Agreed.  Will remove in the next commit.